src/Entities/Chunk.php

```diff
@@ -0,0 +1,386 @@
+<?php
+/**
+ * Chunk Entity
+ *
+ * @package juvo\AS_Processor
+ */
+
+namespace juvo\AS_Processor\Entities;
+
+use Exception;
+use juvo\AS_Processor\DB;
+
+/**
+ * The chunk entity class
+ */
+class Chunk
+{
+    use DB;
+
+    /**
+     * @var int
+     */
+    private int $chunk_id;
+
+    /**
+     * @var int|null
+     */
+    private ?int $action_id = null;
+
+    /**
+     * @var string|null
+     */
+    private ?string $name = null;
+
+    /**
+     * @var string|null
+     */
+    private ?string $group = null;
+
+    /**
+     * @var string|null
+     */
+    private ?string $status = null;
```

Comment:
Add ENUM to specific the status and always use thtat enum. Increas php version as needed